src/lib/ggpo/network/udp_proto.cpp

```diff
@@ -296,8 +296,13 @@ UdpProtocol::HandlesMsg(sockaddr_in &from,
    if (!_udp) {
       return false;
    }
+#ifdef _WIN32
```

Comment:
Windows has a `#define s_addr  S_un.S_addr` in `WinSock2.h`, so I bet you can just use the `else` version on all platforms.